candle-core/Cargo.toml

```diff
@@ -27,6 +27,7 @@ rand_distr = { workspace = true }
 safetensors = { workspace = true }
 thiserror = { workspace = true }
 zip = { workspace = true }
+ureq = { workspace = true }
```

Comment:
Sorry I didn't see earlier, but can we remove `ureq` from core dependency ? I failed to see it earlier. 

I can take care of it

Reply:
sure, it will become candle::Error::Wrapped